ksqldb-engine/src/test/java/io/confluent/ksql/integration/SecureIntegrationTest.java

```diff
@@ -112,6 +114,10 @@
       .outerRule(Retry.of(3, ZooKeeperClientException.class, 3, TimeUnit.SECONDS))
       .around(TEST_HARNESS);
 
+
+  @Rule
+  public final Timeout timeout = Timeout.seconds(30);
```

Comment:
Curious: why the addition of a timeout to this test file in particular? Seems unrelated to the changes in this PR?

Reply:
Is unrelated, but given this test deadlocked locally, I thought I'd just add the timeout.